### PR TITLE
add ImageHash term

### PR DESCRIPTION
**Pull request by victormartinezjurado**
add imagehash term

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -528,6 +538,54 @@ <h3>image</h2>
...
+              <td>
+imageHash
+              </td>
+            </tr>
+            <tr>
+              <td>
+URL
+              </td>
+              <td>
+https://schema.org#imageHash
```
Comment (davidlehn) [inline index.html]: This doesn't appear to be defined at schema.org.

Comment (victormartinezjurado) [reply index.html]: Hi David, you are right, imageHash is not defined at schema.org. Would it be correct to  change the url pointing to https://w3id.org/citizenship#imageHash ? 
I've noticed there are other urls that are not defined at schema.org like https://schema.org#birthCountry . Should it be replaced with https://schema.org/birthPlace

Comment (msporny) [reply index.html]: > Would it be correct to change the url pointing to https://w3id.org/citizenship#imageHash ?

Yes, that would be the right thing to do, although there is an issue w/ imageHash. Will explain in a separate comment.

Comment (msporny) [reply index.html]: > I've noticed there are other urls that are not defined at schema.org like https://schema.org#birthCountry . Should it be replaced with https://schema.org/birthPlace

Yes, the spec is wrong, it should refer to https://schema.org/birthPlace.

File: index.html [hunk 2]
```diff
@@ -260,6 +260,16 @@ <h2>The Citizenship Vocabulary</h2>
 A representative image of the individual.
             </td>
           </tr>
+          <tr>
+            <td>
+imageHash
```
Comment (msporny) [inline index.html:266]: This is the wrong thing to do. We should be using hashlinks if we want to link to content-addressed content external to a Verifiable Credential:

https://tools.ietf.org/html/draft-sporny-hashlink-04

An alternative is to use a data URL (which is what we're doing for now):

https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs

**Review by msporny — CHANGES_REQUESTED** [review]
We shouldn't merge this, it's the wrong solution to the problem.

File: index.html (hunk 2 above)
Comment (victormartinezjurado) [reply index.html:266]: Manu, I have updated the branch with your corrections

Comment (msporny) [reply index.html:266]: Hmm, what I'm saying is that imageHash is completely unnecessary in both scenarios, 1) Where you are embedding the image, or 2) When you are referring to the image using a hashlink. This is what a hashlink to the image could look like:

```
  "image": "hl:zQmWvQxTqbG2Z9HPJgG57jjwR154cKhbtJenbyYTWkjgF3e:zuh8iaLobXC8g9tfma1CSTtYBakXeSTkHrYA5hmD4F7dCLw8XYwZ1GWyJ3zwF"
```

or

```
  "image": "https://images.example.com/people/23849723947?hl=zQmWvQxTqbG2Z9HPJgG57jjwR154cKhbtJenbyYTWkjgF3e"
```

**Comment by msporny** [comment]
Can you retarget this PR to the `main` branch, please, @victormartinez-work and fix the merge conflicts?

File: index.html [hunk 3]
```diff
@@ -528,6 +539,54 @@ <h3>image</h2>
         </table>
 
       </section>
+      <section class="normative">
+        <h3>imageHash</h2>
```
Comment (msporny) [inline index.html:543]: Hmm, could you rename this to just `image`?

**Comment by davidlehn** [comment]
Just a note that this was only closed due to the target branch going away.  There are various solutions for similar issues, and it's still an area of development.  For VCs, the v2 spec has some guidance on such issues:
https://www.w3.org/TR/vc-data-model-2.0/#integrity-of-related-resources

**Pull request closed without merging**